.actions/setup_tools.py

```diff
@@ -394,74 +178,63 @@ def parse_version_from_file(pkg_root: str) -> str:
     return ver
 
 
-def prune_duplicate_lines(body):
-    body_ = []
-    # drop duplicated lines
-    for ln in body:
-        if ln.lstrip() not in body_ or ln.lstrip() in (")", ""):
-            body_.append(ln)
-    return body_
-
-
-def create_meta_package(src_folder: str, pkg_name: str = "pytorch_lightning", lit_name: str = "pytorch"):
-    """Parse the real python package and for each module create a mirroe version with repalcing all function and
-    class implementations by cross-imports to the true package.
+def _replace_imports_in_file(lines: List[str], pkg_lut: Dict[str, str]) -> List[str]:
+    """REplace imports of standlone package to lightning.
+
+    >>> lns = ["lightning_app",
+    ...        "delete_cloud_lightning_apps",
+    ...        "from lightning_app import",
+    ...        "lightning_apps = []",
+    ...        "lightning_app is ours",
+    ...        "def _lightning_app():",
+    ...        ":class:`~lightning_app.core.flow.LightningFlow`"]
+    >>> from pprint import pprint
+    >>> pprint(_replace_imports_in_file(lns, {"app": "lightning_app"}))
+    ['lightning.app',
+     'delete_cloud_lightning_apps',
+     'from lightning.app import',
+     'lightning_apps = []',
+     'lightning.app is ours',
+     'def _lightning_app():',
+     ':class:`~lightning.app.core.flow.LightningFlow`']
+    """
+    for n2, n1 in pkg_lut.items():
+        for i, ln in enumerate(lines):
+            lines[i] = re.sub(rf"([^_]|^){n1}([^_\w]|$)", rf"\1lightning.{n2}\2", ln)
+    return lines
 
-    As validation run in termnal: `flake8 src/lightning/ --ignore E402,F401,E501`
 
-    >>> create_meta_package(os.path.join(_PROJECT_ROOT, "src"))
-    """
+def copy_adjusted_modules(src_folder: str, pkg_name: str, lit_name: str, pkg_lut: dict) -> None:
```

Comment:
couldn't we just import this from the assistant.py? Should be exactly the same.

Reply:
assistant is not distributed with the package, so in contrast, we may think about importing `setup_tools` in assistant...

Reply:
fine with me as well. Just saying that we don't want the same logic twice unless absolutely necessary.

Reply:
Agree, would you be fine to leave this unification in separate PR?
```suggestion
# TODO: unify usage with assistant function, such that import this function in there
def copy_adjusted_modules(src_folder: str, pkg_name: str, lit_name: str, pkg_lut: dict) -> None:
```

Reply:
definitely!

Reply:
> The change should be quite simple, and it avoids adding all this code, to the best of my knowledge, this would be the diff

I think it is the issue with the diff tool as we removed 6 functions completely and wrote two new, but the diff is trying to find some "random parity and then it shows wild diff :/"

Reply:
My point is that doing what @justusschock proposed at the top should be easy, just need to load the assistant and call it as I showed above. Do you see any other issues?

Reply:
I'll resolve to unblock